java/ql/integration-tests/all-platforms/kotlin/extractor_information_kotlin1/ExtractorInformation.expected

```diff
@@ -0,0 +1,7 @@
+| Number of files with extension jar | 1 |
+| Number of files with extension kt | 1 |
+| Number of lines of code | 2 |
+| Number of lines of code with extension kt | 2 |
+| Total number of lines | 3 |
+| Total number of lines with extension kt | 3 |
+| Uses Kotlin 2: false | 1 |
```

Comment:
Would it be possible to add another integration test to check `Uses Kotlin 2: true`? Maybe with a gradle config with `id 'org.jetbrains.kotlin.jvm' version '1.9...'`.

Reply:
We already have tests that fail if your Kotlin version isn't `ci_version`, so I've added an integration test that uses the system `kotlin`. Currently we get loads of diagnostics due to K2 support not being complete, but it does give us `| Uses Kotlin 2: true | 1 |` which is the main thing.